shark/examples/shark_inference/stable_diffusion/opt_params.py

```diff
@@ -23,7 +23,8 @@ def get_unet():
     if args.precision == "fp16":
         if args.use_tuned:
             bucket = "gs://shark_tank/vivian"
-            model_name = "unet_1dec_fp16_tuned"
+            if args.version == "v2.1base":
```

Comment:
We can leave the v1.4 version as it is. `if args.version == "v1.4": model_name = "un.....1dec..."`.

Reply:
Done